pkg/apis/componentconfig/types.go

```diff
@@ -250,7 +250,7 @@ type KubeletConfiguration struct {
 	// streamingConnectionIdleTimeout is the maximum time a streaming connection
 	// can be idle before the connection is automatically closed.
 	StreamingConnectionIdleTimeout metav1.Duration
-	// nodeStatusUpdateFrequency is the frequency that kubelet posts node
+	// NodeStatusUpdateFrequency is the frequency that kubelet posts node
```

Comment:
Our convention for API types is to have the comments match the json field names, which begin with a lowercase letter.